autopts/client.py

```diff
@@ -1436,7 +1436,7 @@ def init_iutctl(self, args):
         autoprojects.iutctl.init(args)
 
     def setup_project_pixits(self, ptses):
-        setup_project_pixits(ptses)
+        setup_project_pixits(self.file_paths, ptses)
```

Comment:
`make_error_txt()` doesn't do much now, so let's write directly to the error file to avoid the import from autopts.bot module.
```suggestion
        try:
            setup_project_pixits(ptses)
        except xmlrpc.client.Fault:
            with open(self.file_paths['ERROR_TXT_FILE'], "w") as f:
                f.write('Bad Pixit Exception')
            raise
```

Reply:
After second thought, if we could handle all exceptions in bot layer in this way:
```python
try:
    stats = self.run_tests()
except:
    with open(self.file_paths['ERROR_TXT_FILE'], "w") as f:
        f.write(traceback.format_exc())
        raise
finally:
    release_device(self.args.tty_file)
```

Reply:
It is here https://github.com/auto-pts/auto-pts/blob/b2f98b6155b371a46a49e53d005886d7ac683624/autopts/bot/common.py#L501-L504

Reply:
thanks, applied your idea